Cargo.toml

```diff
@@ -1,6 +1,11 @@
 [workspace]
 resolver = "2"
-members = ["nym-vpn-lib", "nym-vpn-cli", "nym-vpnd"]
+members = [
+    "crates/nym-gateway-directory",
```

Comment:
Is the `crates` naming usual for rust projects? `common`, like in the mixnet repo, sounds more familiar, but I might just be melancholic here :D